From d16f263c9cec47016918d3aef76494ddb9d58280 Mon Sep 17 00:00:00 2001
From: matt <mleung975@gmail.com>
Date: Tue, 19 Sep 2023 17:38:53 -0700
Subject: [PATCH] changes

---
 go.mod      | 82 ++++++++++++++++++++++++++---------------------------
 metadata.mk |  2 +-
 2 files changed, 42 insertions(+), 42 deletions(-)

diff --git a/go.mod b/go.mod
index 0eac71dee..1253d6de3 100644
--- a/go.mod
+++ b/go.mod
@@ -84,16 +84,16 @@ require (
 	gopkg.in/go-playground/validator.v9 v9.27.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.6
-	k8s.io/apiextensions-apiserver v0.26.6
-	k8s.io/apimachinery v0.26.6
-	k8s.io/apiserver v0.26.6
-	k8s.io/client-go v0.26.6
-	k8s.io/code-generator v0.26.6
-	k8s.io/component-base v0.26.6
+	k8s.io/api v0.26.8
+	k8s.io/apiextensions-apiserver v0.26.8
+	k8s.io/apimachinery v0.26.8
+	k8s.io/apiserver v0.26.8
+	k8s.io/client-go v0.26.8
+	k8s.io/code-generator v0.26.8
+	k8s.io/component-base v0.26.8
 	k8s.io/klog/v2 v2.80.1
 	k8s.io/kube-openapi v0.0.0-20230303024457-afdc3dddf62d
-	k8s.io/kubernetes v1.26.6
+	k8s.io/kubernetes v1.26.8
 	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749
 	modernc.org/memory v1.5.0
 	sigs.k8s.io/kind v0.11.1
@@ -256,19 +256,19 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/warnings.v0 v0.1.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/cloud-provider v0.26.6 // indirect
-	k8s.io/component-helpers v0.26.6 // indirect
+	k8s.io/cloud-provider v0.26.8 // indirect
+	k8s.io/component-helpers v0.26.8 // indirect
 	k8s.io/cri-api v0.0.0 // indirect
-	k8s.io/csi-translation-lib v0.26.6 // indirect
-	k8s.io/dynamic-resource-allocation v0.26.6 // indirect
+	k8s.io/csi-translation-lib v0.26.8 // indirect
+	k8s.io/dynamic-resource-allocation v0.26.8 // indirect
 	k8s.io/gengo v0.0.0-20220902162205-c0856e24416d // indirect
-	k8s.io/kms v0.26.6 // indirect
+	k8s.io/kms v0.26.8 // indirect
 	k8s.io/kube-proxy v0.0.0 // indirect
 	k8s.io/kube-scheduler v0.0.0 // indirect
 	k8s.io/kubectl v0.0.0 // indirect
-	k8s.io/kubelet v0.26.6 // indirect
+	k8s.io/kubelet v0.26.8 // indirect
 	k8s.io/legacy-cloud-providers v0.0.0 // indirect
-	k8s.io/mount-utils v0.26.6 // indirect
+	k8s.io/mount-utils v0.26.8 // indirect
 	k8s.io/pod-security-admission v0.0.0 // indirect
 	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.37 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
@@ -286,34 +286,34 @@ replace (
 
 	github.com/projectcalico/api => ./api
 
-	k8s.io/api => k8s.io/api v0.26.6
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.6
-	k8s.io/apimachinery => k8s.io/apimachinery v0.26.6
-	k8s.io/apiserver => k8s.io/apiserver v0.26.6
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.6
-	k8s.io/client-go => k8s.io/client-go v0.26.6
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.6
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.6
-	k8s.io/code-generator => k8s.io/code-generator v0.26.6
-	k8s.io/component-base => k8s.io/component-base v0.26.6
-	k8s.io/component-helpers => k8s.io/component-helpers v0.26.6
-	k8s.io/controller-manager => k8s.io/controller-manager v0.26.6
-	k8s.io/cri-api => k8s.io/cri-api v0.26.6
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.6
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.6
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.6
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.6
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.6
-	k8s.io/kubectl => k8s.io/kubectl v0.26.6
-	k8s.io/kubelet => k8s.io/kubelet v0.26.6
+	k8s.io/api => k8s.io/api v0.26.8
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.26.8
+	k8s.io/apiserver => k8s.io/apiserver v0.26.8
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.8
+	k8s.io/client-go => k8s.io/client-go v0.26.8
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.8
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.8
+	k8s.io/code-generator => k8s.io/code-generator v0.26.8
+	k8s.io/component-base => k8s.io/component-base v0.26.8
+	k8s.io/component-helpers => k8s.io/component-helpers v0.26.8
+	k8s.io/controller-manager => k8s.io/controller-manager v0.26.8
+	k8s.io/cri-api => k8s.io/cri-api v0.26.8
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.8
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.8
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.8
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.8
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.8
+	k8s.io/kubectl => k8s.io/kubectl v0.26.8
+	k8s.io/kubelet => k8s.io/kubelet v0.26.8
 
 	// Need replacements for all the k8s subsidiary projects that are pulled in indirectly because
 	// the kubernets repo pulls them in via a replacement to its own vendored copies, which doesn't work for
 	// transient imports.
-	k8s.io/kubernetes => k8s.io/kubernetes v1.26.6
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.6
-	k8s.io/metrics => k8s.io/metrics v0.26.6
-	k8s.io/mount-utils => k8s.io/mount-utils v0.26.6
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.26.6
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.26.6
+	k8s.io/kubernetes => k8s.io/kubernetes v1.26.8
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.8
+	k8s.io/metrics => k8s.io/metrics v0.26.8
+	k8s.io/mount-utils => k8s.io/mount-utils v0.26.8
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.26.8
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.26.8
 )
diff --git a/metadata.mk b/metadata.mk
index e1e14cc2b..bdcd8dcff 100644
--- a/metadata.mk
+++ b/metadata.mk
@@ -8,7 +8,7 @@ GO_BUILD_VER = v0.88
 # Version of Kubernetes to use for tests.
 K8S_VERSION     = v1.26.6
 # This is used for bitnami/kubectl and kubectl binary release.
-KUBECTL_VERSION = v1.24.14
+KUBECTL_VERSION = v1.26.8
 
 # Version of various tools used in the build and tests.
 COREDNS_VERSION=1.5.2
-- 
2.42.2

